backend/app/routes.py

```diff
@@ -247,13 +254,16 @@ def aggregate_travel_times(start_node, end_node, start_time, end_time, start_dat
                     "node_start": start_node,
                     "node_end": end_node,
                     "time_range": f"[{start_time},{end_time})", # ints
-                    "date_range": f"[{start_date},{end_date})" # 'YYYY-MM-DD'
+                    "date_range": f"[{start_date},{end_date})", # 'YYYY-MM-DD'
+                    "incl_holiday": holiday,
+                    "dow_list": tuple(dow_list)

```

Comment:
I think this would just split between every character of the string?

Reply:
What happened to the code I pushed in https://github.com/CityofToronto/bdit_tt_request_app/pull/52/commits/677d96c3f2454644401a8b2395266ecbcf4994e2?

Reply:
I guess because it's turning it into an array instead of a tuple. Why not try something like 
```python
"dow_list": tuple( [ int(x) for x in dow_list.split('') ] )
```

Then we can split by a comma too making the input easier to read. Tuple wasn't really the right function for this task anyway. Better to use the `split` method.